### PR TITLE
[RFR] first poc for display better pieces on screen

### PR DESCRIPTION
**Pull request by JulienMattiussi**
https://trello.com/c/6Lygvpr6
pieces are now displayed using 4 caracteristics on the console :
- the symbole
- the text color
- the back color
- the hole in symbol
-> Graphics seams ok

https://trello.com/c/gxYOUxXF
Added in PR cause it was only a particularity on docker fixed in one line

### CODE REVIEW

**Comment by fzaninotto** [comment]
Please add screenshot in PR description (same for every PR which changes the UI)

File: src/game/game.py [hunk 1]
```diff
@@ -44,14 +44,14 @@ def start(self):
     def parse_state_from_args(self, argv):
         if 'quarto.py' in argv[0]:
             argv.pop(0)
-
+        parameter = ""
         try:
             opts, args = getopt.getopt(argv, "", ["state="])
             for opt, arg in opts:
                 if opt == "--state":
                     parameter = arg
```
Comment (jdemangeon) [inline src/game/game.py]: Why this double assignation ?

File: src/game/game.py [hunk 2]
```diff
@@ -44,14 +44,14 @@ def start(self):
...
-            parameter = json.loads(parameter)
-        except (getopt.GetoptError, UnboundLocalError, json.decoder.JSONDecodeError, json.JSONDecodeError):
+                    parameter = json.loads(parameter)
+        except (getopt.GetoptError, json.decoder.JSONDecodeError, json.JSONDecodeError):
             parameter = ""
```
Comment (jdemangeon) [inline src/game/game.py]: Useless because parameter is initialized before try

Comment (JulienMattiussi) [reply src/game/game.py]: In fact, I was affraid that json.loads() could overwrite the variable even if an exception was triggered, but you'are right

**Review by jdemangeon — APPROVED** [review]

File: src/game/ui.py [hunk 3]
```diff
@@ -118,7 +118,6 @@ def selected_player_to_string(self, player_name, selected):
 
     def clear_terminal(self):
         subprocess.call(["printf", "'\033c'"])
-        os.system('cls' if os.name == 'nt' else 'clear')
```
Comment (fzaninotto) [inline src/game/ui.py]: closes #12

**Comment by JulienMattiussi** [comment]
Closes #10

**Pull request merged**